doc/source/f2py/windows/index.rst

```diff
@@ -0,0 +1,95 @@
+.. _f2py-windows:
+
+=================
+F2PY and Windows
+=================
+
+.. note::
+
+	F2PY support for Windows is not at par with Linux support, and 
+	OS specific flags can be seen via ``python -m numpy.f2py``
+
+Broadly speaking there are two issues working with F2PY on Windows, the lack of actively
+developed FOSS Fortran compilers, and the linking issues related to the C
+runtime library for building Python-C extensions.
+
+The focus of this chapter is to establish a guideline for developing and
+extending Fortran modules for Python natively, via F2PY on Windows. 
+For this document we will asume the following basic tools:
+
+- The IDE being considered is the community supported `Microsoft Visual Studio Code`_
+- The shell environment is assumed to be `Powershell 7.x`_
+- Python 3.9 from `the Microsoft Store`_ and this can be tested with
+``Get-Command python.exe`` resolving to
+``C:\Users\$USERNAME\AppData\Local\Microsoft\WindowsApps\python.exe``
+- The  Microsoft Visual C++ (MSVC) toolset
+
+With this baseline configuration, we will further consider a configuration
+matrix as follows:
+
+.. _table-f2py-winsup-mat:
+
+.. table:: Support matrix, exe implies a Windows installer 
+
++----------------------+--------------------+------------+
+| **Fortran Compiler** | **C/C++ Compiler** | **Source** |
++======================+====================+============+
+| GFortran             | MSVC               | MSYS2/exe  |
++----------------------+--------------------+------------+
+| GFortran             | GCC                | MSYS2      |
++----------------------+--------------------+------------+
+| GFortran             | GCC                | WSL        |
++----------------------+--------------------+------------+
+| GFortran             | GCC                | Cygwin     |
++----------------------+--------------------+------------+
+| Classic Flang        | MSVC               | Source     |
++----------------------+--------------------+------------+
+| Intel Fortran        | MSVC               | exe        |
++----------------------+--------------------+------------+
+| Anaconda GFortran    | Anaconda GCC       | exe        |
++----------------------+--------------------+------------+
+
+`This MSYS2 document`_ covers details of differences between MSYS2 and Cygwin,
+but broadly speaking, MSYS2 is geared towards building native Windows but Cygwin
+is closer to providing a `more complete POSIX environment`_. Since MSVC is a
+core component of the Windows setup, its installation and the setup for the
+Powershell environment are described below.
```

Comment:
```suggestion
`This MSYS2 document`_ covers details of differences between MSYS2 and Cygwin.
Broadly speaking, MSYS2 is geared towards building native Windows, while Cygwin
is closer to providing a `more complete POSIX environment`_. Since MSVC is a
core component of the Windows setup, its installation and the setup for the
Powershell environment are described below.
```